### DISCUSSION TITLE
Home Assistant cards

### DISCUSSION THREAD

**Discussion by Nerdboy2** [rbaron/b-parasite#184] [General]
Hello everyone
I am looking for inspiration for how to represent all the sensors in the HA dashboard. 
What cards are you all using? Any customs cards that you would like to recommend? 

Thanks!

**Comment by fsalfner-dev** [comment]
Hey,

I have built 5 parasite sensors and have built my HA dashboard as follows:

1. A "map" view showing where each sensor is using a `picture-elements` card with `state-badge` elements showing soil moisture placed on a floorplan of each room. When tapping on a state badge, the dashboard navigates to the corresponding detail view.
2. A "detail view" for each of the parasites showing a history graph of soil moisture and the other sensor values

The "map view" looks as follows:

<img src="https://github.com/rbaron/b-parasite/assets/112203157/fa6ec8e7-7abf-447c-bc8f-62806aa09ac8" width=300px />

And the "detail view" looks as follows:

<img src="https://github.com/rbaron/b-parasite/assets/112203157/2b15663b-0315-4e37-86e0-a97491b866c9" width=300px />
